handout/features/README.md

```diff
@@ -3,7 +3,7 @@
 ![ES6/TypeScript](../images/es6-typescript.png "ES6 and TypeScript")
 
 The language we usually call "JavaScript" is formally known as "EcmaScript".
-The new version of JavaScript, "EcmaScript 6" or "ES6", offers a number of new features that extend the power of the language.
-ES6 is not widely supported in today's browsers, so it needs to be transpiled to ES5. 
-You can choose between several transpilers, but we'll be using TypeScript, 
+The new version of JavaScript, "EcmaScript 2015" or "ES6", offers a number of new features that extend the power of the language.
+ES6 is not widely supported in today's browsers, so it needs to be transpiled to ES5.
```

Comment:
ES6 actually is widely supported in modern browsers; it is poorly supported in older browsers. See: http://kangax.github.io/compat-table/es6/